tests/utils.go

```diff
@@ -100,3 +103,80 @@ func GetKubeClientFromRESTConfig(config *rest.Config) (*kubernetes.Clientset, er
 
 	return coreClient, nil
 }
+
+func CreatePVC(namespace string, name string, size string) {
+	client, err := GetKubeClient()
+	PanicOnError(err)
+	_, err = client.CoreV1().PersistentVolumeClaims(namespace).Create(newPVC(name, size))
+	PanicOnError(err)
+}
+
+func DeletePVC(namespace string, name string) {
+	client, err := GetKubeClient()
+	PanicOnError(err)
+	err = client.CoreV1().PersistentVolumeClaims(namespace).Delete(name, nil)
+	PanicOnError(err)
+}
+
+func newPVC(name string, size string) *k8sv1.PersistentVolumeClaim {
+	quantity, err := resource.ParseQuantity(size)
+	PanicOnError(err)
+
+	return &k8sv1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+		Spec: k8sv1.PersistentVolumeClaimSpec{
+			AccessModes: []k8sv1.PersistentVolumeAccessMode{k8sv1.ReadWriteOnce},
+			Resources: k8sv1.ResourceRequirements{
+				Requests: k8sv1.ResourceList{
+					"storage": quantity,
+				},
+			},
+		},
+	}
+}
+
+func RunPodWithPVC(namespace string, podName string, pvcName string, cmd string) {
+	client, err := GetKubeClient()
```

Comment:
almost every func has to do this. k8s e2e tests use the previously mentioned _framework_ struct to hold the client fields, and often the client is passed to the target func to avoid getting the client each call.